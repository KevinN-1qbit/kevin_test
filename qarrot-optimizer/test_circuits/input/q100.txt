Rotate 2: IZIIIII
Rotate 2: IXIIIII
Rotate 2: IZIIIII
Rotate 1: IIIIIIZ
Rotate 1: IIIIIZI
Rotate 1: IZIIIII
Rotate 2: IIIIIZX
Rotate -2: IIIIIZI
Rotate -2: IIIIIIX
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate 2: IXIIIIZ
Rotate -2: IIIIIIZ
Rotate -2: IXIIIII
Rotate -1: IIIIIZI
Rotate 2: IIIIIXZ
Rotate -2: IIIIIIZ
Rotate -2: IIIIIXI
Rotate -1: IIIIIIZ
Rotate -1: IIIIIZI
Rotate 1: IZIIIII
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate 2: IXIIIIZ
Rotate -2: IIIIIIZ
Rotate -2: IXIIIII
Rotate 2: IIIIIZX
Rotate -2: IIIIIZI
Rotate -2: IIIIIIX
Rotate 2: IZIIIII
Rotate 2: IXIIIII
Rotate 2: IZIIIII
Rotate 2: ZIIIIII
Rotate 2: XIIIIII
Rotate 2: ZIIIIII
Rotate 1: IZIIIII
Rotate 1: IIIIIZI
Rotate 1: ZIIIIII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: ZIIIIXI
Rotate -2: ZIIIIII
Rotate -2: IIIIIXI
Rotate 2: XZIIIII
Rotate -2: IZIIIII
Rotate -2: XIIIIII
Rotate -1: IIIIIZI
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate -1: IZIIIII
Rotate -1: IIIIIZI
Rotate 1: ZIIIIII
Rotate 2: ZIIIIXI
Rotate -2: ZIIIIII
Rotate -2: IIIIIXI
Rotate 2: XZIIIII
Rotate -2: IZIIIII
Rotate -2: XIIIIII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: ZIIIIII
Rotate 2: XIIIIII
Rotate 2: ZIIIIII
Rotate 2: IIZIIII
Rotate 2: IIXIIII
Rotate 2: IIZIIII
Rotate 1: IZIIIII
Rotate 1: IIIIIZI
Rotate 1: IIZIIII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIZIIXI
Rotate -2: IIZIIII
Rotate -2: IIIIIXI
Rotate 2: IZXIIII
Rotate -2: IZIIIII
Rotate -2: IIXIIII
Rotate -1: IIIIIZI
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate -1: IZIIIII
Rotate -1: IIIIIZI
Rotate 1: IIZIIII
Rotate 2: IIZIIXI
Rotate -2: IIZIIII
Rotate -2: IIIIIXI
Rotate 2: IZXIIII
Rotate -2: IZIIIII
Rotate -2: IIXIIII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIZIIII
Rotate 2: IIXIIII
Rotate 2: IIZIIII
Rotate 2: IIIZIII
Rotate 2: IIIXIII
Rotate 2: IIIZIII
Rotate 1: IZIIIII
Rotate 1: IIIIIZI
Rotate 1: IIIZIII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIIZIXI
Rotate -2: IIIZIII
Rotate -2: IIIIIXI
Rotate 2: IZIXIII
Rotate -2: IZIIIII
Rotate -2: IIIXIII
Rotate -1: IIIIIZI
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate -1: IZIIIII
Rotate -1: IIIIIZI
Rotate 1: IIIZIII
Rotate 2: IIIZIXI
Rotate -2: IIIZIII
Rotate -2: IIIIIXI
Rotate 2: IZIXIII
Rotate -2: IZIIIII
Rotate -2: IIIXIII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIIZIII
Rotate 2: IIIXIII
Rotate 2: IIIZIII
Rotate 2: IIIIZII
Rotate 2: IIIIXII
Rotate 2: IIIIZII
Rotate 1: IZIIIII
Rotate 1: IIIIIZI
Rotate 1: IIIIZII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIIIZXI
Rotate -2: IIIIZII
Rotate -2: IIIIIXI
Rotate 2: IZIIXII
Rotate -2: IZIIIII
Rotate -2: IIIIXII
Rotate -1: IIIIIZI
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate -1: IZIIIII
Rotate -1: IIIIIZI
Rotate 1: IIIIZII
Rotate 2: IIIIZXI
Rotate -2: IIIIZII
Rotate -2: IIIIIXI
Rotate 2: IZIIXII
Rotate -2: IZIIIII
Rotate -2: IIIIXII
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIIIZII
Rotate 2: IIIIXII
Rotate 2: IIIIZII
Rotate 2: IIIIIIZ
Rotate 2: IIIIIIX
Rotate 2: IIIIIIZ
Rotate 1: IZIIIII
Rotate 1: IIIIIZI
Rotate 1: IIIIIIZ
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIIIIXZ
Rotate -2: IIIIIIZ
Rotate -2: IIIIIXI
Rotate 2: IZIIIIX
Rotate -2: IZIIIII
Rotate -2: IIIIIIX
Rotate -1: IIIIIZI
Rotate 2: IZIIIXI
Rotate -2: IZIIIII
Rotate -2: IIIIIXI
Rotate -1: IZIIIII
Rotate -1: IIIIIZI
Rotate 1: IIIIIIZ
Rotate 2: IIIIIXZ
Rotate -2: IIIIIIZ
Rotate -2: IIIIIXI
Rotate 2: IZIIIIX
Rotate -2: IZIIIII
Rotate -2: IIIIIIX
Rotate 2: IXIIIZI
Rotate -2: IIIIIZI
Rotate -2: IXIIIII
Rotate 2: IIIIIIZ
Rotate 2: IIIIIIX
Rotate 2: IIIIIIZ
