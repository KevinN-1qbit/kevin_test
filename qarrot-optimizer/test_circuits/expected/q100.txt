Rotate 1: IIIIIIZ
Rotate 1: IXIIIII
Rotate -1: IXIIIZI
Rotate -1: IIIIIZZ
Rotate -1: IXIIIIZ
Rotate 1: IXIIIZZ
Rotate 1: XIIIIII
Rotate -1: XIIIIZI
Rotate 1: IIXIIII
Rotate -1: IIXIIZI
Rotate 1: IIIXIII
Rotate -1: IIIXIZI
Rotate 1: IIIIXII
Rotate -1: IIIIXZI
Rotate -1: XZIIIII
Rotate 1: XZIIIZI
Rotate -1: IZXIIII
Rotate 1: IZXIIZI
Rotate -1: IZIXIII
Rotate 1: IZIXIZI
Rotate -1: IZIIXII
Rotate 1: IZIIXZI
Rotate 1: IZIIIII
Rotate 1: IIIIIIX
Rotate -1: IIIIIZX
Rotate -1: IZIIIZI
Rotate -1: IZIIIIX
Rotate 1: IZIIIZX
Measure +: ZIIIIII
Measure +: IZIIIII
Measure +: IIZIIII
Measure +: IIIZIII
Measure +: IIIIZII
Measure +: IIIIIZI
Measure +: IIIIIIZ
